releasenotes/notes/propagation_behavior_extract-3d16765cfd07485b.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    propagation: Introduces the environment variable ``DD_TRACE_PROPAGATION_BEHAVIOR_EXTRACT`` 
```

Comment:
We should also note that this PR introduces support for propagating baggage when trace headers are not received (we could also move this fix to it's own PR).